drf/accounts/views.py

```diff
@@ -44,18 +38,30 @@ def post(self, request, *args, **kwargs):
             user_obj = User.objects.get(Q(username__iexact=username) |
```

Comment:
You could use something like: https://docs.djangoproject.com/en/2.1/topics/http/shortcuts/#get-object-or-404